src/web/client/common/localStore.ts

```diff
@@ -23,56 +25,62 @@ export interface IPowerPlatformExtensionContext {
     entity: string;
     rootDirectory: vscode.Uri;
     saveDataMap: Map<string, SaveEntityDetails>,
-    defaultFileUri: vscode.Uri // This will default to home page or current page in multifile scenario
+    defaultFileUri: vscode.Uri, // This will default to home page or current page in multifile scenario
+    contextSet: boolean,
+    currentSchema: string
 }
 
 class PowerPlatformExtensionContextManager {
```

Comment:
Can we match files names and the class in the files, helps understanding/writing code easier?

Reply:
Also considering this is a context manager it should be a singleton?

Reply:
yeah, it is a singleton pattern implemented here. Will take file renames separately. Out of scope for this PR.